packages/vsphere/_dev/deploy/docker/Dockerfile

```
@@ -0,0 +1,8 @@
+ARG VSPHERE_GOLANG_VERSION
+FROM golang:${VSPHERE_GOLANG_VERSION}-alpine
+
+RUN apk add --no-cache curl git
+RUN go get -u github.com/vmware/govmomi/vcsim
```

It'd be nice to fix here the version of vcsim used, and it is probably better to use `go install`.
```suggestion
RUN go install github.com/vmware/govmomi/vcsim@${VSPHERE_VCSIM_VERSION}
```

---

Not having the version, will ensure having the latest vcsim in the docker container. 
Won't that help us in being updated ?

---

If you don't fix the version, it may happen that a change in vcsim breaks your builds, affecting unrelated PRs. With a fixed version you will have more reproducible builds.

---

Hmm, correct. Made the changes